dist/js/Alert.d.ts

```diff
@@ -0,0 +1,19 @@
+export default class SKAlert {
+    constructor({ title, text, buttons }: {
+        title: any;
+        text: any;
+        buttons: any;
```

Comment:
Since those files are `.js`, tsc uses `any` for all of these. We'll convert all of those to `.ts` and assign them the correct types. Probably a separate task.